src/Http/RequestFactory.php

```diff
@@ -218,4 +231,38 @@ public function createHttpRequest()
 		return new Request($url, NULL, $post, $files, $cookies, $headers, $method, $remoteAddr, $remoteHost, $rawBodyCallback);
 	}
 
+	/**
+	 * @param bool $usingTrustedProxy
```

Comment:
Where did you get these information? I was following the [coding standards](https://nette.org/en/coding-standard). Omitting parameter name breaks PhpStorm and is not recommended by phpDocumentator - are you really sure that we want that?